packages/mui-material/src/Backdrop/Backdrop.test.js

```diff
@@ -19,6 +19,9 @@ describe('<Backdrop />', () => {
     slots: {
       root: {
         expectedClassName: classes.root,
+        transition: {
+          testWithElement: null,
+        },
       },
     },
```

Comment:
Slots should be at the same level:

```suggestion
    slots: {
      root: {
        expectedClassName: classes.root,
      },
      transition: {
          testWithElement: null,
        },
    },
```